lib/beacon/live_admin/web.ex

```diff
@@ -62,6 +62,7 @@ defmodule Beacon.LiveAdmin.Web do
 
   defp html_helpers do
     quote do
+      use Beacon.LiveAdmin.StationUI.HTML
```

Comment:
We can import directly from the StationUI dep once it gets published

Reply:
you mean, just inject the js/css and call the component modules remotely?

Reply:
Exactly, unless we want to have fine grained control over updates on those components.

Reply:
Or depending on how UXD want to manage changes, either use stock components or customize. Either way we have some options here but for now only vendor will work anyway.